src/client/src/ui/spotTile/views/spotTileView.jsx

```diff
@@ -26,64 +21,60 @@ export default class SpotTileView extends ViewBase {
-        <div className='currency-pair-title'>
-          <div className='container-controls'>
-            <i className='tearoff-trigger glyphicon glyphicon-new-window pull-right'
-               onClick={() => router.publishEvent(this.props.modelId, 'popOutTile', {})}/>
-            <i className='glyphicon glyphicon-stats pull-right'
-               onClick={() => router.publishEvent(this.props.modelId, 'toggleSparkLineChart', {})}/>
-          </div>
-          <span>{model.tileTitle}</span>
-          <i className='fa fa-plug animated infinite fadeIn'/>
+        <div className='spot-tile__container'>
+          <span className='spot-tile__symbol'>{model.tileTitle} <i className='spot-tile__icon--disconnected fa fa-plug animated infinite fadeIn'/></span>
+          <span className='spot-tile__execution-label'>Executing</span>
+          <i className='spot-tile__icon--tearoff fa fa-square-o'
+             onClick={() => router.publishEvent(this.props.modelId, 'popOutTile', {})}/>
+          {notification}
+          {priceComponents}
+            <NotionalInput
+              className={notionalInputClass}
+              notional={model.notional}
+              onChange={(notional) => router.publishEvent(this.props.modelId, 'notionalChanged', { notional:notional })}
+              currencyPair={model.currencyPair}
+            />
         </div>
-        {notification}
-        {priceComponents}
-        <div className='clearfix'></div>
-        <NotionalInput
-          className={notionalInputClass}
-          notional={model.notional}
-          onChange={(notional) => router.publishEvent(this.props.modelId, 'notionalChanged', { notional:notional })}
-          currencyPair={model.currencyPair}
-        />
-        <div className='clearfix'></div>
-        {sparklineChart}
       </div>
     );
   }
 
   _createPriceComponents() {
     let model:SpotTileModel = this.state.model;
-    let pricingContainerClass = classnames('currency-pair-actions', {'hide': model.hasNotification});
+    let pricingContainerClass = classnames({'hide': model.hasNotification});
     if (model.currentSpotPrice === null) {
       return null;
     }
     return (
       <div className={pricingContainerClass}>
         <PriceButton
+          className='spot-tile__price--bid'
           direction={Direction.Sell}
           onExecute={() => router.publishEvent(this.props.modelId, 'executeTrade', { direction:Direction.Sell })}
           rate={model.currentSpotPrice.bid}
         />
-        <PriceMovementIndicator
-          priceMovementType={model.currentSpotPrice.priceMovementType}
-          spread={model.currentSpotPrice.spread}
-        />
+        <div className='spot-tile__price-movement'>
+          <PriceMovementIndicator
+            priceMovementType={model.currentSpotPrice.priceMovementType}
+            spread={model.currentSpotPrice.spread}
+          />
+        </div>
         <PriceButton
+          className='spot-tile__price--ask'
```

Comment:
Yeah, double dash is used to identify the modifier part. We use single dash is the name of the element or block is made of multiple words.